twoopstracker/twoops/models.py

```diff
@@ -156,3 +156,22 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class AccountGroup(TimestampedModelMixin):
+    name = models.CharField(max_length=255, help_text=_("Name of Account Group"))
+    owner = models.ForeignKey(
+        "UserProfile", on_delete=models.CASCADE, help_text="Owner of the group"
+    )
+    accounts_lists = models.ManyToManyField(
```

Comment:
1. If possible, lets call this just `lists` or use the full name `twitter_accounts_lists`. `account_lists` is confusing i.e. which accounts? `UserAccount` or `TwitterAccount`?
2. What are we calling the `related_name`?

Reply:
>What are we calling the related_name?

We are calling it `teams` so it will be a `twitter_account_list.teams` to get all teams this list belongs to.